Can't seem to build for Android

When compiling a project for Android using the current go-text/typesetting I get the following error:

```
# github.com/benoitkugler/textlayout/harfbuzz
../../vendor/github.com/benoitkugler/textlayout/harfbuzz/ot_use.go:307:53: postBaseFlags64 (untyped int constant 1024345505792) overflows int
```

--- a/go.mod
+++ b/go.mod
@@ -3,7 +3,7 @@ module github.com/go-text/typesetting
 go 1.15
 
 require (
-	github.com/benoitkugler/textlayout v0.0.5
+	github.com/benoitkugler/textlayout v0.1.3
 	golang.org/x/image v0.0.0-20210607152325-775e3b0c77b9
 	golang.org/x/text v0.3.7 // indirect
 )

